src/main/authentication/cftIdamAuthentication.ts

```diff
@@ -0,0 +1,63 @@
+import {FRONTEND_URL} from '../helpers/envUrls';
+import jwt_decode from 'jwt-decode';
+import config = require('config');
+import process from 'process';
+import {cftIdamTokenApi} from '../resources/requests/utils/axiosConfig';
+import {Logger} from '@hmcts/nodejs-logging';
+
+const querystring = require('querystring');
+
+const acceptedRoles = ['IDAM_ADMIN_USER'];
+
+let cftIdamClientSecret;
+
+if(process.env.CFT_IDAM_CLIENT_SECRET) {
+  cftIdamClientSecret = process.env.CFT_IDAM_CLIENT_SECRET;
+} else {
+  cftIdamClientSecret = config.get('secrets.pip-ss-kv.CFT_IDAM_CLIENT_SECRET') as string;
+}
+
+//Added logging temporarily to check if it is being picked up
+const logger = Logger.getLogger('cftIdamAuthentication');
+logger.info(cftIdamClientSecret.substring(0,2));
```

Comment:
Can this be removed now since we have confirmed the client secret is working, or do you still need the log for something else?

Reply:
Nah this can be removed, done :)